### PR TITLE
[RHDX-140] Embedded content assembly

### PR DESCRIPTION
**Pull request by staceymosier**
### JIRA Issue Link
[RHDX-140](https://projects.engineering.redhat.com/browse/RHDX-140)

### Verification Process
View content utilizing the Embedded Content assembly type, or /assembly/[id]/view
Verify that Slideshare.net and Youtube renders with the standard iframe embed code.
All other values should render as a hyperlink.

Youtube: https://developer-preview-3461.ext.us-west.dc.preprod.paas.redhat.com/che-custom-stacks/
Slideshare: Log in to view this unpublished example article: https://developer-preview-3461.ext.us-west.dc.preprod.paas.redhat.com/this-is-the-article-title/?type=article

### CODE REVIEW

**Comment by staceymosier** [comment]
rebuild this please

**Comment by jordanpagewhite** [comment]
The YouTube video looks great. The Slideshare actual embed looks great to me, but the content/data is not fetching from Slideshare for me at least: https://developer-preview-3461.ext.us-west.dc.preprod.paas.redhat.com/this-is-the-article-title/?type=article Have you experienced this? Do you know if perhaps it is rate throttling our domain or something like that? @staceymosier

**Comment by staceymosier** [comment]
This is currently pending:

1. Alistairs review of the responsive iframe solution vs flex display
2. Input on if visual styles "no padding" is intended to remove _margins_ instead of padding, or if these visual styles are only supported on content that has background images.

**Comment by robpblake** [comment]
rebuild this please

**Comment by robpblake** [comment]
rebuild this please

**Comment by staceymosier** [comment]
rebuild this please

**Comment by alistairmc** [comment]
rebuild this please

**Comment by robpblake** [comment]
rebuild this please

**Comment by staceymosier** [comment]
@alistairmc  I noticed in the changes some added PHP in the diff that shouldn't have been there.  It's already in the file further below. Must have been inherited in a update from upstream/master.  This is now ready for merging as long as the latest commit is approved.

**Comment by jordanpagewhite** [comment]
rebuild this please

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme [hunk 1]
```diff
@@ -406,6 +406,25 @@ function rhdp2_preprocess_assembly(&$variables) {
     $variables['attributes']['class'][] = 'rhd-m-video-background';
   }
 
+  // Embedded Content assembly type.
+  if ($type == 'embedded_content') {
+    $field_url = $assembly->get('field_url')->getValue();
+    $embed_url = $field_url[0]['uri'];
```
Comment (jordanpagewhite) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhdp2.theme]: Note to self: Fine since the field is required

**Comment by jordanpagewhite** [comment]
rebuild this please

**Comment by jordanpagewhite** [comment]
rebuild this please

**Comment by robcoffman** [comment]
rebuild this please

**Comment by robcoffman** [comment]
rebuild this please

**Comment by robcoffman** [comment]
rebuild this please

**Comment by robpblake** [comment]
rebuild this please

**Comment by robpblake** [comment]
rebuild this please

**Pull request merged**